index.js

```diff
@@ -5,12 +5,14 @@ var React = require('react');
 var EXT_REGEX = new RegExp('\\.jsx$');
 var DEFAULTS = {
     doctype: '<!DOCTYPE html>',
-    renderMethod: 'renderToStaticMarkup',
-    layoutMethod: 'renderToStaticMarkup',
+    viewRenderMethod: 'renderToStaticMarkup',
+    layoutRenderMethod: 'renderToStaticMarkup',
+    layoutKeyword: 'content',
```

Comment:
Yes. I was disappointed to realize that the view engine couldn't get this option passed to it. However, if the user did provide a `layoutKeyword` to their server config, they'd need to pass this option to our view engine in order for it to continue to work.

And because they are directly related I think it's appropriate to keep the property names the same. Yes, let's document it very clearly.